### PR TITLE
All the bugs fix PR

### PR DESCRIPTION
**Pull request by smellyshovel**
## Description

Replace old inputs with the new ones. Fix VaColorPaletteCustom's inner alignment. Fix TreeView presentation. Fix Spinners presentation.

Fix `dashboard -> setup remote connections -> overview` broken alignment.

Fix Login and Signup pages layouts and add Login/Signup entry to the sidebar.

Fix circle progress bars presentation.

Fix Form Elements' inputs.

Fix VaDatePicker calendar-icon.

Fix Spacing Playground.

Fix ColorPresentation alignment.

Fix Chips presentataion.

Fix Notifications layout, replace old inputs for toast parameters with new ones, fix VaNotification display on narrow screens.

[Bugs](https://github.com/epicmaxco/vuestic-admin/issues/616) №1, №3, №4, №5, №13, №14, №25, №30, №31, №32, №39.

## Markup:

<details>

```vue
// Your code
```
</details>

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: src/components/ui/spinners/Spinners.vue [hunk 1]
```diff
@@ -1,83 +1,69 @@
 <template>
-  <div class="row spinners">
-    <div class="flex md12 xs12">
-      <va-card
-        :title="$t('spinners.title')"
-        class="spinners__widget"
+<div class="spinners">
```
[the remark this answers is not in the document]
Comment (smellyshovel) [reply src/components/ui/spinners/Spinners.vue]: VaColorPaletteSimple relates to another bug report and will be considered separately.

P.S. Do it like what?

File: src/vuestic-theme/vuestic-components/va-select/VaSelect.demo.vue [hunk 2]
```diff
@@ -79,6 +79,13 @@
         :options="defaultSelect.options"
       />
     </VbCard>
+    <VbCard title="No clear" style="width: 400px;">
+      <va-select
+        v-model="defaultSelect.value"
+        :options="defaultSelect.options"
+        no-clear
```
Comment (nastassiadanilova) [inline src/vuestic-theme/vuestic-components/va-select/VaSelect.demo.vue]: maybe, it's better to name this clearable and make it true by default.
In any case, add a new prop to the doc

Comment (jean-moldovan) [reply src/vuestic-theme/vuestic-components/va-select/VaSelect.demo.vue]: Everywhere I've seen (quasar, vuetify), `clearable` is false by default.

File: src/vuestic-theme/vuestic-components/va-navbar/VaNavbar.vue [hunk 3]
```diff
@@ -116,7 +114,6 @@ $nav-border-side-width: 3.1875rem;
     height: 1rem;
     margin: auto;
     z-index: 2;
```
Comment (smellyshovel) [inline src/vuestic-theme/vuestic-components/va-navbar/VaNavbar.vue]: We most probably don't need this line anymore as well. I previously fixed the same bug with this line, but your fix seems to be more robust.

**Pull request merged**